doc/vaffle.txt

```diff
@@ -234,14 +234,14 @@ version, but not supported.
                 time.
 
 *g:vaffle_open_selected_split_position*
-                (Default: 'topleft')
-                Possible values: 'topleft' or 'rightbelow'
+                (Default: '')
+                Possible values: empty (''), 'topleft' or 'rightbelow'
```

Comment:
I think `''` is enough (`empty('')` -> `''`)

Reply:
Revised